src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/BuffersTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using Azure.Core.Buffers;
+using Azure.Core.Testing;
+using NUnit.Framework;
+using System;
+
+namespace Azure.Core.Tests
+{
+    public class BuffersTests
+    {
+        TestPool<byte> pool = new TestPool<byte>();
+
+        [Test]
+        public void EmptyBuilder() {
+            pool.ClearDiagnostics();
+            var sequence = new Sequence<byte>(pool);
```

Comment:
I think all these tests for Spequence should be removed. Sequence used to be a type in Azure.Core but then I just moved it to the test project.

Reply:
Will https://github.com/Azure/azure-sdk-for-net/issues/5219 cover this as well?